testing_and_setup/compass/ocean/hurricane/USDEQU60at15cr5rr1WD/sandy/config_forward.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<config case="forward">
```

Comment:
@pwolfram, should this be renamed `forward_RK` to be consistent with the other cases?

Reply:
Also changed in driver script too...